MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,51 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
```

Comment:
The enclosed code has no direct dependence on Alpaka. Why does it need to be enclosed in `ALPAKA_ACCELERATOR_NAMESPACE`? Or could that namespace level be removed (after which this header should be placed directly in `MagneticField/ParametrizedEngine/interface/`)?

If the `ALPAKA_ACCELERATOR_NAMESPACE` is kept, this header should
```cpp
#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
```

Reply:
Following the discussion with @namapane, I think that eventually we will want an `ESProducer` that builds this object based on the actual magnetic field, and copies it to the devices.

Reply:
> Following the discussion with @namapane, I think that eventually we will want an `ESProducer` that builds this object based on the actual magnetic field, and copies it to the devices.

That sounds good, but could the use of `ALPAKA_ACCELERATOR_NAMESPACE` and `alpaka` subdirectory be left to that time (and applied only to those components that need it)?

Reply:
OK: https://github.com/cms-sw/cmssw/pull/44360/commits/1230b32772a589c18f243f740c7a4fbbc43bdbda